include/grpc/grpc_security.h

```diff
@@ -931,6 +931,9 @@ typedef struct grpc_tls_custom_verification_check_request {
      * grpc_security_constants.h.
      * TODO(ZhenLian): Consider fixing this in the future. */
     const char* peer_cert_full_chain;
+    /* The verified root cert subject.
+     * This value will only be filled if the validation was successful */
```

Comment:
nit: I'd suggest we say "cryptographic peer certificate verification" instead of "validation".

Rationale: I think it's important to disambiguate between the "validator" that we're modifying here (which occurs after the TLS handshake) and the cryptographic verification (which happens during the TLS handshake).

Reply:
Changed